packages/@aws-cdk/aws-rds/lib/database-secret.ts

```diff
@@ -55,5 +66,20 @@ export class DatabaseSecret extends secretsmanager.Secret {
         excludeCharacters: props.excludeCharacters ?? DEFAULT_PASSWORD_EXCLUDE_CHARS,
       },
     });
+
+    if (props.overrideLogicalId) {
+      const hash = crypto.createHash('md5');
+      hash.update(JSON.stringify({
+        path: this.node.path, // make it unique
+        // Use here the options that influence the password generation.
+        // If at some point we add other password customization options
+        // they sould be added here below (e.g. `passwordLength`).
+        excludeCharacters: props.excludeCharacters,
```

Comment:
So `excludeCharacters` is actually optional. Which means this object will have `undefined` there as the default. Is that a mistake? Should the default value be filled here? (See line 66 above)